components/readwise/sources/new-documents/new-documents.mjs

```diff
@@ -41,7 +41,7 @@ export default {
       const { results: events } = await this.readwise.listDocuments({
         params,
       });
-      for (const event of events) {
+      for (const event of events.splice(0, 100)) {
```

Comment:
I think you mean `.slice` here to get the first 100 items in the array, `.splice` would remove them

Reply:
Oh yes. I will change it. (Tough in fact it works in this case😂, `splice` will return the removed items)